com.ibm.streamsx.inet/impl/java/src/com/ibm/streamsx/inet/http/HTTPPostOper.java

```diff
@@ -157,6 +158,11 @@ public void setMaxRetries(int val) {
 	public void setRetryDelay(double val) {
 		this.retryDelay = val;
 	}
+	@Parameter(optional=true, description="Optional parameter specifies amount of time (in seconds) that the operator waits for the connection for to be established.")
```

Comment:
Need details in the description of what the default time out is.

Did this pull-request change the default timeout, there were comments about it being 60 seconds but it's not really clear?